mungegithub/mungers/bulk-lgtm.go

```diff
@@ -343,47 +347,49 @@ func (b *BulkLGTM) ServeLogin(res http.ResponseWriter, req *http.Request) {
 	http.Redirect(res, req, url, http.StatusTemporaryRedirect)
 }
 
-func (b *BulkLGTM) ServeCallback(res http.ResponseWriter, req *http.Request) {
-	state := req.FormValue("state")
-	if state != b.oauthStateString {
-		glog.Errorf("invalid oauth state, expected '%s', got '%s'\n", b.oauthStateString, state)
-		http.Redirect(res, req, "/bulkprs/auth", http.StatusTemporaryRedirect)
-		return
-	}
+func (b *BulkLGTM) ServeCallback(secureCookie bool, cookieDur time.Duration) func(res http.ResponseWriter, req *http.Request) {
```

Comment:
Just like with the githubOauthConfig struct there would be a data race on the `disableSecureCookie` and `cookieDuration` fields because they are used in a gouroutine besides the main one.